src/uws_client.c

```diff
@@ -1093,8 +1095,12 @@ static void on_underlying_io_bytes_received(void* context, const unsigned char*
             case UWS_STATE_WAITING_FOR_UPGRADE_RESPONSE:
             {
                 /* Codes_SRS_UWS_CLIENT_01_378: [ When on_underlying_io_bytes_received is called while the uws is OPENING, the received bytes shall be accumulated in order to attempt parsing the WebSocket Upgrade response. ]*/
-                unsigned char* new_received_bytes = (unsigned char*)realloc(uws_client->stream_buffer, uws_client->stream_buffer_count + size + 1);
-                if (new_received_bytes == NULL)
+                unsigned char* new_received_bytes;
+
+                //size_t realloc_size = uws_client->stream_buffer_count + size + 1; **using safe int**
```

Comment:
Should remove?

Reply:
No, I want it there to show the intended math using safe macros